src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+
+        Colors[] colorValues = Colors.values();
+        Random rand = new Random();
```

Comment:
Creating a new instance of `Random` every time the method is called can be inefficient. Consider making `rand` a private static final field of the class.